### ISSUE TITLE
Should helm template also render templates/tests directory?

### ISSUE THREAD

**Issue by invidian** [helm/helm#6483]
Currently when `helm template` command is invoked to render all manifests, also manifests from `templates/tests` are rendered, which might be unwanted. I'm currently downloading the helm charts locally, renders them with my values, to perform further modifications on them with kustomize, which seems to be a valid use case.

I think it would be useful to have a flag in `helm template` which would exclude tests templates from rendering.

Output of `helm version`:
```
$ helm version --client
Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
```

**Comment by mattfarina** [comment]
I can see why someone would want this. I would suggest it as a feature for Helm v3.

**Comment by bacongobbler** [comment]
Implemented in #6444
